apis/management.cattle.io/v3/schema/schema.go

```diff
@@ -149,14 +149,26 @@ func authzTypes(schemas *types.Schemas) *types.Schemas {
 			&m.Embed{Field: "status"}).
 		AddMapperForType(&Version, v3.GlobalRole{}, m.DisplayName{}).
 		AddMapperForType(&Version, v3.RoleTemplate{}, m.DisplayName{}).
+		AddMapperForType(&Version,
+			v3.PodSecurityPolicyTemplateProjectBinding{},
+			&mapper.NamespaceIDMapper{}).
 		AddMapperForType(&Version, v3.ProjectRoleTemplateBinding{},
 			&mapper.NamespaceIDMapper{},
 		).
-		MustImport(&Version, v3.Project{}).
+		MustImport(&Version, v3.SetPodSecurityPolicyTemplateInput{}).
+		MustImportAndCustomize(&Version, v3.Project{}, func(schema *types.Schema) {
+			schema.ResourceActions = map[string]types.Action{
+				"setpodsecuritypolicytemplate": {
+					Input:  "setPodSecurityPolicyTemplateInput",
+					Output: "project",
+				},
+			}
+		}).
 		MustImport(&Version, v3.GlobalRole{}).
 		MustImport(&Version, v3.GlobalRoleBinding{}).
 		MustImport(&Version, v3.RoleTemplate{}).
 		MustImport(&Version, v3.PodSecurityPolicyTemplate{}).
+		MustImport(&Version, v3.PodSecurityPolicyTemplateProjectBinding{}).
```

Comment:
Right now, the PSPTB resource is probably exposed as an api endpoint at /v3 (you should be able to see it in your browser). This also means that by default, CRUD operations will be exposed directly in the API. We want to prevent that. I think we can do so by customizing the schema and setting CollectionMethods and ResourceMethods to empty slices. (These default to values like POST, GET, PUT, DELETE)